core/src/components/divider/test/basic/divider.e2e.ts

```diff
@@ -0,0 +1,40 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+configs({ directions: ['ltr', 'rtl'], modes: ['ios', 'md', 'ionic-md'] }).forEach(({ config, screenshot, title }) => {
+  test.describe(title('divider: rendering'), () => {
+    test('should not have visual regressions', async ({ page }) => {
+      await page.setContent(
+        `
+          <div id="container">
+            top
+            <ion-divider></ion-divider>
+            bottom
+          </div>
+        `,
+        config
+      );
+
+      const container = page.locator('#container');
+
+      await expect(container).toHaveScreenshot(screenshot(`divider-basic-default`));
+    });
+
+    test('should not have visual regressions when inset is enabled', async ({ page }) => {
```

Comment:
If you want to keep the rtl tests for inset since it applies to left/right then you can move it to a separate config like this:

```ts
configs({ modes: ['ios', 'md', 'ionic-md'] }).forEach(({ config, screenshot, title }) => {
```

If you leave `directions` out of the config it will run for both `ltr` and `rtl` 🙂